### ISSUE TITLE
Powershell install script returns 404

### ISSUE THREAD

**Issue by luxus** [twpayne/chezmoi#1123]
## Describe the bug
```
PowerShell 7.2.0-preview.4
Copyright (c) Microsoft Corporation.

https://aka.ms/powershell
Type 'help' to get help.

PS C:\Users\luxus> (iwr https://git.io/chezmoi.ps1).Content | powershell -c -
checking GitHub for tag latest
found tag v2.0.7 for latest
found version 2.0.7 for latest/windows/amd64
An error occurred while installing: Exception calling "DownloadFile" with "2" argument(s): "The remote server returned an error: (404) Not Found."
PS C:\Users\luxus>

```
## To reproduce
Open a windows terminal
enter the ` (iwr https://git.io/chezmoi.ps1).Content | powershell -c -`

## Expected behavior
Installation of Chezmoi
## Additional context
windows build 21343.1000
PowerShell 7.2.0-preview.4

**Comment by twpayne** [comment]
It looks like `iwr` is not following the redirect from https://git.io/chezmoi.ps1 to https://raw.githubusercontent.com/twpayne/chezmoi/master/assets/scripts/install.ps1.

I had a quick look at the [`iwr` documentation](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/invoke-webrequest?view=powershell-7.1) but didn't see any mention of redirects.

@zb140 you know this area the best, do you know what could be happening here?

**Comment by zb140** [comment]
Actually, I'm pretty sure it's just because the checksums file got renamed and the script is still looking for the old name.  We know it's not the initial redirect because the first 3 lines of output are actually from `install.ps1` so it must be executing.

I can fix it in a few hours after work, but I'm pretty sure just changing [this](https://github.com/twpayne/chezmoi/blob/master/assets/scripts/install.ps1#L273) to `checksums.txt` will do the trick.  I noticed the change in the bash install script the other day but for whatever reason it didn't click that the Powershell one would probably need updating too.  Sorry about that!

**Comment by zb140** [comment]
@persalmi fixed this for us in #1125. Thanks, all!

**Comment by persalmi** [comment]
Missed that there was a registered issue for it...
